### PR TITLE
Fix the remove group modal

### PR DESCRIPTION
**Pull request by lgalis**
Refactored the RemoveGroup modal component.
 WIP for now  to add a test.

Before- fetch group loop
After:
Screenshot from 2019-09-20 17-46-02

Update: Added a placeholder while the group data is loaded:
Screenshot from 2019-09-24 15-26-11

### CODE REVIEW

**Comment by lgalis** [comment]
@rvsia - would you be able to review this?

File: src/smart-components/group/remove-group-modal.js [hunk 1]
```diff
@@ -9,23 +9,19 @@ import { fetchGroups, fetchGroup, removeGroup } from '../../redux/actions/group-
...
   removeGroup,
+  group,
+  isLoading,
   fetchGroup,
-  fetchGroups,
-  groupId,
-  group
+  fetchGroups
 }) => {
+
```
Comment (rvsia) [inline src/smart-components/group/remove-group-modal.js]: unnecessary empty line

File: src/test/group/__snapshots__/remove-group-modal.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<RemoveGroupModal /> should render correctly 1`] = `
```
Comment (rvsia) [inline src/test/group/__snapshots__/remove-group-modal.test.js.snap]: This snapshot is not very useful, use `shallow(...).dive()` (render one more layer of the component) or mount (to render everything)

File: src/smart-components/group/remove-group-modal.js (hunk 1 above)
Comment (lgalis) [reply src/smart-components/group/remove-group-modal.js]: removed

File: src/test/group/__snapshots__/remove-group-modal.test.js.snap (hunk 2 above)
Comment (lgalis) [reply src/test/group/__snapshots__/remove-group-modal.test.js.snap]: agree - removed this test

**Comment by lgalis** [comment]
@rvsia  - there is a value in the modal name - but is not the correct one, there is only one load. I do need to load the name, to not use stale data. 
I added a placeholder until the name is loaded.

**Comment by rvsia** [comment]
@lgalis thanks for adressing the issues!

**Comment by lgalis** [comment]
@rvsia - Thank you for the review!

**Comment by lgalis** [comment]
@gmcculloug or @syncrou - would you be able to merge this approved PR?

**Pull request merged**